kamu-core/Cargo.toml

```diff
@@ -56,6 +56,7 @@ sha3 = "0.10"
 aws-config = { version = "0.55" }
 aws-sdk-s3 = { version = "0.28" }
 aws-smithy-http = { version = "0.55", features = ["rt-tokio"] }
+aws-credential-types = "*"
```

Comment:
Please specify a version - I'm making `cargo deny` rules more strict to reject the wildcard dependencies.

Reply:
Done